docs/docs/examples/model-inference/Stable-Diffusion-CKPT-Inference/index.md

```diff
@@ -101,119 +96,102 @@ Next, upload the image to the registry. This can be done by using the Docker hub
 docker push <hub-user>/<repo-name>:<tag>
 ```
 
-In our case
+So in our case, the command will look like this:
```

Comment:
This was used a few lines up, perhaps there's another way of saying it?

Reply:
Changed wording